runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -219,7 +219,8 @@ local function on_line_impl(self, buf, line)
         a.nvim_buf_set_extmark(buf, ns, start_row, start_col,
                                { end_line = end_row, end_col = end_col,
                                  hl_group = hl,
-                                 ephemeral = true
+                                 ephemeral = true,
+                                 priority = 1 -- Smallest possible priority (bc we are the base)
```

Comment:
I don't think this is good. It should be possible for a plugin to say that their colors should be _below_ treesitter.

Reply:
Then what value do you want me to put in here ? Something like `100` would make tree-sitter below default value but still leaves room below

Reply:
yes, 100 sounds fine :)

Reply:
Done